components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -642,7 +644,27 @@ public enum ErrorMessages {
                 "attribute for organization with ID: %s."),
         ERROR_CODE_ERROR_LISTING_ORGANIZATIONS_DISCOVERY_ATTRIBUTES("65122", "Error while listing " +
                 "organizations discovery attributes.", "Server encountered an error while fetching discovery " +
-                "attributes for organizations.");
+                "attributes for organizations."),
+        ERROR_CODE_ERROR_CREATE_ORGANIZATION_USER_ASSOCIATION("65123", "Unable to create organization user " +
+                "association.", "Server encountered an error while creating organization user association " +
+                "for user %s."),
```

Comment:
```suggestion
                "for user with ID: %s."),
```
Let's follow the same pattern to other error codes as well

Reply:
Fixed by - https://github.com/wso2/identity-organization-management-core/pull/86/commits/771160a62abd4d31df387491a8cfcf67704c451d